ui/src/translations/fr.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Bienvenu",
+  "name": "Nom",
+  "cpu_capacity": "CPU Capacité",
```

Comment:
Hum, that doesn't make sense in French.
I don't know what we want to express here so I can't propose a better translation (do you mean "Nombre de CPU"?).